api/src/api/opportunities_v1/opportunity_schemas.py

```diff
@@ -535,3 +535,39 @@ class OpportunitySearchResponseV1Schema(AbstractResponseSchema, PaginationMixinS
         OpportunityFacetV1Schema(),
         metadata={"description": "Counts of filter/facet values in the full response"},
     )
+
+
+class UserV1Schema(Schema):
+    user_id = fields.String(
+        metadata={
+            "description": "The internal ID of a user",
+            "example": "861a0148-cf2c-432b-b0b3-690016299ab1",
+        }
+    )
+    email = fields.String(
+        metadata={
+            "description": "The email address returned from Oauth2 provider",
+            "example": "js@gmail.com",
+        }
+    )
+    external_user_type = fields.String(
+        metadata={
```

Comment:
This should be an enum pointing to the ExternalUserType that was created

Reply:
Updated this to enum `ExternalUserType`